R/utils.R

```diff
@@ -28,6 +70,276 @@ colfun <- function() {
   ))
 }
 
+
+#' Control Fonts
+#'
+#' @description `control_fonts()` calculates font-sizes depending on output size
+#'
+#' @param base_font_size (unit)\cr Font-size of normal paragraph text.
+#'
+#'   By default is 9pt.
+#'
+#'   By editing this value, all the other parameters get updated too.
+#'
+#' @param h1 (unit)\cr Font-size of title of the graph,
+#'   dependent of `base_font_size`. By default, 12pt.
+#'
+#' @param h2 (unit)\cr Font-size of subtitle of the graph,
+#'   dependent of `base_font_size`. By default, 10pt.
+#'
+#' @param label (unit)\cr Font-size of text that's called outside the ggplot
+#'   theme. It should render at the same font size as base_font_size
+#'
+#' @details It returns a list with the font sizes of:
+#'
+#' * **p** - normal text elements within the ggplot theme.
+#'
+#' * **h1** - titles.
+#'
+#' * **h2** - subtitles.
+#'
+#' * **label** - for text elements outside the ggplot theme. If you want to add
+#' an annotation or a label to your plot, use the font size `label`. By default,
+#' it renders at 10pt.
+#'
+#' * **rel** - for legend elements.
+#'
+#' @export
+#' @examples
+#' control_fonts(base_font_size = 10)
+control_fonts <- function(base_font_size = 9,
+                          h1 = 12,
+                          h2 = 10,
+                          label = base_font_size + 1) {
+  rel <- 7.253 / 9
```

Comment:
Should it be `rel <- 7.253 / base_font_size` here, or it does not change with base_font_size?

Reply:
Thanks Chen - line updated.